dags/sentinel-2_data_transfer.py

```diff
@@ -38,57 +43,167 @@ def extract_src_key(src_url):
     :param src_url: Full http path of the object
     :return: Object path relative to the base bucket
     """
+
     matches = (re.finditer("/", src_url))
     matches_positions = [match.start() for match in matches]
     start = matches_positions[2] + 1
     return src_url[start:]
 
+def load_stac(s3_hook, stac_json_path):
+    """
+    load the STAC file associated with each scene
+    :param s3_hook: Hook to interact with S3
+    :param stac_json_path: Path to the STAC file on S3
+    :return: Dictionary of the STAC file
+    """
+
+    content_object = s3_hook.get_key(key=extract_src_key(stac_json_path),
+                     bucket_name=default_args['src_bucket_name'])
+    content = content_object.get()['Body'].read().decode('utf-8')
+    data = json.loads(content)
+    return data
+
+def africa_extent():
+    """
+    Load Africa boundaries
+    :return: Shape object of the boundaries
+    """
+
+    boundaries = json.load(open("data/africa-extent.json"))
```

Comment:
Do you need open as well?

Reply:
json.load needs a file object so, yes it needs the open.